File: AccuPay/Core/Utilities/StringUtils.vb
Imports System.Globalization
Imports System.Text.RegularExpressions

Namespace Global.AccuPay.Utils

    Public Class StringUtils

        Private Sub New()
        End Sub

        Public Shared Function ToPascal(text As String) As String

            If text Is Nothing Then Return Nothing

            Dim newText = Regex.Replace(text, "([A-Z])", " $1")

            Dim info = CultureInfo.CurrentCulture.TextInfo
            Return info.ToTitleCase(newText).Replace(" ", String.Empty)
        End Function

        Public Shared Function Strip(text As String) As String

            If text Is Nothing Then Return Nothing

            Return text.Replace(" ", String.Empty)
        End Function

        Public Shared Function Normalize(text As String) As String
            Return text?.Trim()?.ToUpper()
        End Function

    End Class

End Namespace

File: AccuPay.Utilities/StringUtils.cs
using System.Globalization;
using System.Text.RegularExpressions;

namespace AccuPay.Utilities
{
    public static class StringUtils
    {
        public static string ToPascal(string text)
        {
            if (text == null) return null;

            string newText = Regex.Replace(text, "([A-Z])", " $1");

            var info = CultureInfo.CurrentCulture.TextInfo;
            return info.ToTitleCase(newText).Replace(" ", string.Empty);
        }

        public static string Normalize(string text)
        {
            return text?.Trim()?.ToUpper();
        }
    }
}